<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_set_wave</class>
 <widget class="QDialog" name="dialog_set_wave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="spinBox_point1_x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_point1_y">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_point2_y">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_point2_x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_count">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_amplitude">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_wave_len">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_speed">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_step">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Скорость
волны</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Длина
волны</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Амплитуда
волны</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Кривизна</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество участков
разбиения по ОХ и ОУ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Начальная точка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Направление</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_dir_x">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>244</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_dir_y">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>244</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
